tests/appsec/test_traces.py

```diff
@@ -130,6 +130,9 @@ def validate_response_headers(span):
 
     @missing_feature(library="python")
     @bug(context.library < "java@0.93.0")
+    @irrelevant(
```

Comment:
is it because it's not possible to generate a non-web trace on this framework, or because the actual weblog does not generate one ? 

If it's the second choice, we need to skip it using `missing_feature`, like PHP

Reply:
I've fixed in the latest commit.